src/main/java/org/jabref/logic/importer/fetcher/DoiResolution.java

```diff
@@ -140,8 +140,9 @@ public Optional<URL> findFullText(BibEntry entry) throws IOException {
     }
 
     /**
-     * Scan for <meta name="citation_pdf_url">
-     * See https://scholar.google.com/intl/de/scholar/inclusion.html#indexing
+     * Scan for {@code <meta name="citation_pdf_url">}.
+     *
```

Comment:
Checkstyle seems to be complaining that a blank line should separate the block-tag section. But 144 *is* as blank as it can be. Any clue what I'm missing? Of course I can revert this particular change, but is that the point of Checkstyle?

Reply:
I think check style expects a return somewhere in this method and thus checks for the empty lines
https://checkstyle.sourceforge.io/checks/javadoc/requireemptylinebeforeblocktaggroup.html

Reply:
It seems that `@see` does not support external links unless they are wrapped in `<a>`. `@see <a href="url">url</a>`. I forgot about this, because APIdia does accept plain urls in `@see` and links them properly (as this pattern occurs frequently in the wild). So I'll revert the use of `@see` here.